src/sim_init.cc

```diff
@@ -246,6 +267,8 @@ void SimInit::LoadSolverInfo() {
 
   if (solver_name == "greedy") {
     solver = LoadGreedySolver(exclusive_orders, tables);
+  } else if (solver_name == "coin-or") {
+    solver = LoadCoinSolver(exclusive_orders, tables);
```

Comment:
Does the coin solver have any state that evolves over the course of the simulation?  If so, we will need to persist that in simulation snapshots and reload it again too.